backend/lcfs/web/api/document/views.py

```diff
@@ -40,7 +40,7 @@ async def get_all_documents(
     response_model=FileResponseSchema,
     status_code=status.HTTP_201_CREATED,
 )
-@view_handler([RoleEnum.SUPPLIER])
+@view_handler([RoleEnum.SUPPLIER, RoleEnum.ANALYST])
```

Comment:
Just a question, can the supplier upload the files, even after the compliance report is submitted using this endpoint? as this could pose a security threat.

Reply:
There is no possibility for the supplier to upload files when the report is submitted in the Front End. However, I can add a condition in the backend as well.

Reply:
yes, please. Let's add some validations at the backend as well.